docs/user-guide/downward-api/volume/index.md

```diff
@@ -17,7 +17,7 @@ This example assumes you have a Kubernetes cluster installed and running, and th
 
 ### Step One: Create the pod
 
-Use the `docs/user-guide/downward-api/dapi-volume.yaml` file to create a Pod with a  downward API volume which stores pod labels and pod annotations to `/etc/labels` and  `/etc/annotations` respectively.
+Use the `docs/user-guide/downward-api/volume/dapi-volume.yaml` file to create a Pod with a downward API volume which stores pod labels and pod annotations to `/etc/labels` and `/etc/annotations` respectively.
```

Comment:
How about we link this file so that the confusion is over once and for all. Might be especially helpful since this was originally written when there was only one repo.